internal/xerrors/operation.go

```diff
@@ -189,6 +189,9 @@ func (e *operationError) Type() Type {
 }
 
 func (e *operationError) BackoffType() backoff.Type {
+	if IsOperationErrorTransactionLocksInvalidated(e) {
```

Comment:
I think it can creak some customers, who want fast retries and low latency on TLI.

It should be optional behaviour.